test/parser/postgresparser_test.cpp

```diff
@@ -724,5 +724,23 @@ TEST_F(PostgresParserTests, CreateDbTest) {
   delete stmt_list;
 }
 
+TEST_F(PostgresParserTests, CreatePGSQLFuncTest) {
+  std::string query = "CREATE OR REPLACE FUNCTION increment(i integer) RETURNS integer AS $$   BEGIN  RET  j;  END; $$ LANGUAGE plpgsql;";
+
+  auto parser = parser::PostgresParser::GetInstance();
+  auto stmt_list = parser.BuildParseTree(query).release();
+  EXPECT_TRUE(stmt_list->is_valid);
```

Comment:
Yes, we haven't completed the registering of the function yet. Currently working on this part.